pyperf/_command.py

```diff
@@ -23,18 +24,22 @@ def bench_command(command, task, loops):
                         % proc.returncode)
 
     rss = None
+    metadata = {}
     try:
         lines = output.splitlines()
         timing = float(lines[0])
-        if len(lines) >= 2:
-            rss = int(lines[1])
+        rss = int(lines[1])
```

Comment:
Please explain about the parsing logic.
see: https://github.com/psf/pyperf/pull/197/files#r1718097565 :)